tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -843,7 +803,7 @@ module.exports = {
       selector: '#files-list-progress'
     },
     fileActionsButtonInFileRow: {
-      selector: '//button[@aria-label="show-file-actions"]',
+      selector: '//button[@aria-label="Show file actions"]',
```

Comment:
Please introduce a class and select on that instead of the aria label text

Reply:
Done